src/components/Text/Base.js

```diff
@@ -62,25 +76,37 @@ TextBase.propTypes = {
     "summerSky",
     "turquoise"
   ]),
-  size: PropTypes.oneOf(["uno", "hecto", "kilo", "giga", "tera"]),
+  size: PropTypes.oneOf(["mini", "uno", "hecto", "kilo", "giga", "tera"]),
+  responsiveSize: PropTypes.shape({
+    small: PropTypes.oneOf([Object.keys(typography.size)]),
+    medium: PropTypes.oneOf([Object.keys(typography.size)]),
+    large: PropTypes.oneOf([Object.keys(typography.size)])
```

Comment:
I would prefer if you used the same `PropTypes` whitelist used for the `size` prop for these `responsiveSize` keys. The reason is that we do not want `zetta` or `bronto` sized body `Text`, per the `prism` spec.